decidim-proposals/app/cells/decidim/proposals/proposal_m_cell.rb

```diff
@@ -122,6 +122,24 @@ def has_image?
       def resource_image_path
         @resource_image_path ||= has_image? ? model.attachments.find_by("content_type like '%image%'").url : nil
       end
+
+      def cache_hash
+        hash = []
+        hash << "decidim/proposals/proposal_m"
+        hash << model.cache_version
+        hash << model.proposal_votes_count
+        hash << model.endorsements_count
+        hash << Digest::MD5.hexdigest(model.component.settings.to_json)
+        hash << Digest::MD5.hexdigest(resource_image_path) if resource_image_path
+        if current_user
+          hash << current_user.cache_version
+          hash << current_user.follows?(model) ? 1 : 0
+        end
+        hash << Digest::MD5.hexdigest(model.followers.to_json)
+        hash << Digest::MD5.hexdigest(model.coauthorships.map(&:cache_version).to_s)
+
+        hash.join("/")
```

Comment:
Current locale must be taken into account

Reply:
Done!